db/secondbase/structure.sql

```diff
@@ -1035,23 +1044,153 @@ CREATE FUNCTION vulnerabilities_id_seq_view() RETURNS integer
+
+ALTER TEXT SEARCH CONFIGURATION "default"
+    ADD MAPPING FOR hword WITH english_stem;
+
+ALTER TEXT SEARCH CONFIGURATION "default"
+    ADD MAPPING FOR url_path WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION "default"
+    ADD MAPPING FOR file WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION "default"
+    ADD MAPPING FOR "float" WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION "default"
+    ADD MAPPING FOR "int" WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION "default"
+    ADD MAPPING FOR uint WITH simple;
+
+
+--
+-- Name: pg; Type: TEXT SEARCH CONFIGURATION; Schema: public; Owner: -
+--
+
+CREATE TEXT SEARCH CONFIGURATION pg (
+    PARSER = pg_catalog."default" );
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR asciiword WITH english_stem;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR word WITH english_stem;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR numword WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR email WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR url WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR host WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR sfloat WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR version WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR hword_numpart WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR hword_part WITH english_stem;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR hword_asciipart WITH english_stem;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR numhword WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR asciihword WITH english_stem;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR hword WITH english_stem;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR url_path WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR file WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR "float" WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR "int" WITH simple;
+
+ALTER TEXT SEARCH CONFIGURATION pg
+    ADD MAPPING FOR uint WITH simple;
+
+
+--
+-- Name: ohloh; Type: SERVER; Schema: -; Owner: -
 --
 
-CREATE SERVER openhub FOREIGN DATA WRAPPER postgres_fdw OPTIONS (
+CREATE SERVER ohloh FOREIGN DATA WRAPPER postgres_fdw OPTIONS (
     dbname 'ohloh_development',
-    host 'localhost',
+    host 'oh-db01.dc1.lan',
     port '5432'
 );
 
 
 --
--- Name: USER MAPPING ohloh_user SERVER openhub; Type: USER MAPPING; Schema: -; Owner: -
+-- Name: USER MAPPING fisbot_app_stage SERVER ohloh; Type: USER MAPPING; Schema: -; Owner: -
 --
 
-CREATE USER MAPPING FOR ohloh_user SERVER openhub OPTIONS (
-    password 'password',
-    "user" 'ohloh_user'
+CREATE USER MAPPING FOR fisbot_app_stage SERVER ohloh OPTIONS (
+    password 'Qvnp8Q3mih8IJfMv',
+    "user" 'ohloh_ui_app_stage'
```

Comment:
@msk999 we should not expose these credentials, therefore, please do run the migration against your local development database. I've explained to @drubio1989 probably he could brainstorm you along those lines.